packages/modules/create-itwin-viewer/extensions/desktop/basic.ts

```diff
@@ -3,5 +3,8 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
+import { Viewer } from "@itwin/desktop-viewer-react";
+
```

Comment:
just revert changes for these files altogether. No need for these extra spaces

Reply:
do you mean the empty lines between?